README.md

```diff
@@ -46,10 +46,16 @@ Verified on :
 
 	_Same as in previous step, use `~/.profile` on Ubuntu, `~/.zshrc` for Zsh._
 
-    **Fish note**: Instead, copy `~/.jenv/fish/jenv.fish` to `~/.conf/fish/function/jenv.fish`. If you don't have the `export` function, also copy `export.fish`
+    **Fish note**: 
+    Instead, copy `~/.jenv/fish/jenv.fish` to `~/.config/fish/function/jenv.fish`. If you don't have the `export` function, also copy `export.fish`
     ~~~ sh
-        cp ~/.jenv/fish/jenv.fish ~/.conf/fish/function/jenv.fish
-        cp ~/.jenv/fish/export.fish ~/.conf/fish/function/export.fish
+        cp ~/.jenv/fish/jenv.fish ~/.config/fish/function/jenv.fish
+        cp ~/.jenv/fish/export.fish ~/.config/fish/function/export.fish
```

Comment:
The folder should be `functionS` 😄 
I just followed it though and managed to finally make it work, thanks!